docs/testing-summary.html

```
@@ -0,0 +1,354 @@
+<title>Testing Status</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)